### PR TITLE
Issues fix

### PR DESCRIPTION
**Pull request by Akado2009**
This PR:
1. Fixes issue https://github.com/CiscoAI/kubeflow-workflows/issues/49
2. Fixes issue https://github.com/CiscoAI/kubeflow-workflows/issues/51
3. Adds custom port support. See https://github.com/CiscoAI/kubeflow-workflows/issues/52

### CODE REVIEW

**Comment by xyhuang** [comment]
@ramdootp @johnugeorge do you want to have the webapp in the same directory or keep it in a separate place? just checking.

File: tf-mnist/webapp.bash [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+source variables.bash
+
+# define new variables
+TF_MODEL_SERVER_HOST=`kubectl describe pod mnist -n ${NAMESPACE} | grep IP | sed -E 's/IP:[[:space:]]+//'`
+CLIENT_IMAGE=${DOCKER_HUB}/${DOCKER_USERNAME}/${DOCKER_IMAGE}
+MNIST_SERVING_IP=`kubectl -n ${NAMESPACE} get svc/mnist --output=jsonpath={.spec.clusterIP}`
+
+
+# docker authorization
+if [ "${DOCKER_HUB}" = "docker.io" ]
```
Comment (ramdootp) [inline tf-mnist/webapp.bash]: Is this check correct?

File: tf-mnist/variables.bash [hunk 2]
```diff
@@ -32,3 +32,14 @@ DOCKER_BASE_URL=gcr.io/cpsg-ai-demo
 IMAGE=${DOCKER_BASE_URL}/tf-mnist-demo:v1
 #docker build . --no-cache  -f Dockerfile -t ${IMAGE}
 #docker push ${IMAGE}
+
+# Used in portf.bash and webapp.bash
+# If using without an application, source this file before using
+PORT=9000
+export TF_MODEL_SERVER_PORT=${PORT}
+
+# Used in webapp.bash
```
Comment (ramdootp) [inline tf-mnist/variables.bash]: see if CONTAINER_REGISTRY, USERNAME are more appropriate names that can be considered.

Comment (Akado2009) [reply tf-mnist/variables.bash]: Updated

**Pull request merged**